### PR TITLE
created simple example of rps

### PR DESCRIPTION
**Pull request by MrValdez**
## Description

This adds a new example targeted at beginners. This is a simple Rock Paper Scissors game.

The example demonstrates how to apply HTML to an element and how variables can persist over the webpage's lifetime.

For simplicity, the python code is written to be simple to read and understand from a beginner. This is why global variables are used and the check_winner() isn't optimized.

## Checklist

-   [ ] All tests pass locally
-   [ ] I have updated `docs/changelog.md`
-   [ ] I have created documentation for this(if applicable)

### CODE REVIEW

File: examples/rock-paper-scissors.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                        elif results == "P2":
+                            results_output.write("The computer have won!")
+                            p2_score += 1
+                        else:
+                            results_output.write("Tied!")
+
+                        update_score()
+
+                    def check_winner(p1, p2):
+                        # Coding challenge: this are more efficient ways of checking who would win. Can you figure any of them out?
```
Comment (sabry2000) [inline examples/rock-paper-scissors.html:129]:
```suggestion
                        # Coding challenge: there are more efficient ways of checking who would win. Can you figure any of them out?
```

**Review by sabry2000 — APPROVED** [review]
:)

**Comment by fpliger** [comment]
Hi @MrValdez  and thank you for the PR! We've been trying to not add more examples to the core repo because it adds some difficulties for us to maintain them and were hoping to start a different repo to host community examples. 

Unfortunately, that effort slowed down and, eventually, may not be the best option. I just opened a [discussion ](https://github.com/pyscript/pyscript/discussions/1442) for us to decide on that very topic and hope we can decide and make clarity as soon as possible. For now, I'd say the best option is to put this PR on hold until we decide.

Really appreciate your contribution, your interest, and patience!

**Comment by arnabpal16** [comment]
YES IT'S WORKS

**Pull request closed without merging**